src/presentation/controller/addressLookUp/Routing.ts

```diff
@@ -250,11 +250,23 @@ const addressRoutingEnterGeneralPartnerCorrespondenceAddress = {
   }
 };
 
+const addressRoutingConfirmGeneralPartnerCorrespondenceAddress = {
+  previousUrl: url.POSTCODE_GENERAL_PARTNER_PRINCIPAL_OFFICE_ADDRESS_URL,
```

Comment:
I think this should be referencing `POSTCODE_GENERAL_PARTNER_CORRESPONDENCE_ADDRESS_URL`.

Reply:
Updated.